### PR TITLE
Security Guard Job & STC/MailCarrier Spawn and Requirement change

### PR DESCRIPTION
**Pull request by dvir001**
## About the PR
Adding the security guard job (Station Security) for frontier
Added STC spawn and open job on round start
Added MailCarrier open job on round start

## Why / Balance
Required for security split.

## Technical details
Based on security job, yml.

## Media
- [X] I have added screenshots/videos to this PR showcasing its changes ingame,

image

## Breaking changes
N/A

**Changelog**
N/A

:cl: dvir01
- add: NT Added the security guard job to frontier to help SR & STC keep the outpost safe.

### CODE REVIEW

**Comment by dvir001** [comment]
I need someone to tell me what the ``job-description-security-guard`` need to be.

**Comment by ErhardSteinhauer** [comment]
> I need someone to tell me what the `job-description-security-guard` need to be.

"Patrol the empty station halls, whistle simple tunes you heard on radio, jingle your keychain and scurry away at the sight of danger."

**Comment by dvir001** [comment]
All done

File: Resources/Prototypes/Roles/Jobs/departments.yml [hunk 1]
```diff
@@ -89,6 +89,7 @@
   - PrisonGuard ##nyano
   - Prisoner ##nyano
   - Brigmedic
+  - SecurityGuard # Frontier
```
Comment (TsjipTsjip) [inline Resources/Prototypes/Roles/Jobs/departments.yml:92]: Security guard should not show up under NFSD.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Roles/Jobs/departments.yml (hunk 1 above)
Comment (dvir001) [reply Resources/Prototypes/Roles/Jobs/departments.yml:92]: Pick one:
department-Civilian-description = Perform small helpful tasks to keep the station sane.
department-Command-description = Manage the crew and keep them working efficiently.
department-Engineering-description = Keep the station operational.
department-Medical-description = Keep the crew healthy.
department-Security-description = Keep the peace around the sector.
department-Science-description = Research new technologies and dangerous artifacts.
department-Specific-description = Jobs that not all stations have.

We can also add new one and edit them out, and to be honest, we need to at that point.

Comment (TsjipTsjip) [reply Resources/Prototypes/Roles/Jobs/departments.yml:92]: `department-Security-description` would be most appropriate in this case, as that should show up under "Security" in both role selection and on crew manifests. Might need a different locstring tho, specifically mentioning Frontier Outpost.

Comment (dvir001) [reply Resources/Prototypes/Roles/Jobs/departments.yml:92]: Out of this PR scope, Let me know if you want to edit this FTL files yourself and make a new PR.

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**